test-helpers/ReactQuery.tsx

```diff
@@ -36,18 +36,13 @@ export const createTestQueryClient = () => {
         retry: false,
         retryDelay: 0,
         // NB: This prevents CI from hanging endlessly.
-        cacheTime: Infinity
+        gcTime: Infinity
       },
       mutations: {
-        cacheTime: Infinity,
+        gcTime: Infinity,
         retry: false,
         retryDelay: 0
       }
-    },
-    logger: {
```

Comment:
logger options not supported in queryClient options

Reply:
Thanks for getting rid of this.